packages/graphback-codegen-resolvers/src/templates/resolverTemplates.ts

```diff
@@ -20,7 +20,11 @@ export const deleteTemplate = (modelName: string): string => {
 
 export const findAllTemplate = (modelName: string): string => {
   return `(parent, args, context) => {
-      return context.${modelName}.findAll();
+      if (args.offset && args.limit){
+        return context.${modelName}.findMore(args.offset, args.limit);
```

Comment:
Hi
This is not going to work. This is executed at the time of the code generation. We should always use findAll like this
`return context.${modelName}.findAll(args);`

Reply:
Understood